### ISSUE TITLE
[android] ...more button is not clickable in bookmark's list description

### ISSUE THREAD

**Issue by biodranik** [organicmaps/organicmaps#4813]
There are several options that can be done:
1. Expand TextView to see the full description while keeping it scrollable and other bookmarks below accessible.
2. Open the List Settings dialog where the full description is displayed and is editable.

Need to compare the pros and cons of both and choose the best approach.

Bonus question: can HTML be rendered in the bookmark's list description? A separate issue may be created for HTML rendering if necessary.

#4812 should be merged first to see the "...more" button.

**Comment by ANS-UXI** [comment]
How about if we implement approach(1) and on top of that render an edit button after the TextView has expanded?

**Comment by biodranik** [comment]
Rich descriptions will come from imported tracks. WYSIWYG may be an interesting idea for the future, but it will complicate things right now. 

We can create a separate issue about rich bookmarks, tracks and lists descriptions, with the ability to add photos from the phone. Exporting/sharing it will require a lot of work.

**Comment by ANS-UXI** [comment]
Yeah that's true!

**Comment by Saimurugeshwari** [comment]
Hi @biodranik can i contribute to this issue?

**Comment by Shubham212003** [comment]
I want to work on this issues please assign me this issue

**Comment by biodranik** [comment]
We do not assign issues to newcomers. Just start fixing it without an assignment.
